app/Console/Commands/LdapSync.php

```diff
@@ -69,13 +69,13 @@ public function handle()
 
         $results = Ldap::findLdapUsers();
 
-        $ldap_ou_locations = Location::whereNotNull('ldap_ou')->get();
+        $ldap_ou_locations = Location::whereRaw('LENGTH(ldap_ou) > 0')->get();
```

Comment:
What is the reason for this change? We typically try to avoid raw queries to maintain the possibility of supporting additional databases.

Reply:
This accounts for non-null location OUs that have been cleared to an empty string. An alternative would be to change the location edit method to NULL the OU field if it’s zero length. I'll give this a try, as I agree that my current solution isn't good for portability.